packages/shader-lab/src/ShaderLab.ts

```diff
@@ -0,0 +1,12 @@
+import { IShaderInfo, IShaderLab } from "./interface";
```

Comment:
Should use engine-design package IShaderLab interface

Reply:
fixed